db/schema.rb

```diff
@@ -12,21 +12,21 @@
 
 ActiveRecord::Schema.define(version: 2020_12_10_084736) do
 
-  create_table "analytics", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+  create_table "analytics", charset: "utf8", force: :cascade do |t|
```

Comment:
What exactly is this changing?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/550#discussion_r551576556

Reply:
https://github.com/rails/rails/pull/39365

Reply:
Good; I'm VERY glad they changed this.